test-suite-jakarta-inject-bean-import/src/test/java/io/micronaut/jakartainject/tck/beanimport/BeanImportTck.java

```diff
@@ -12,8 +12,8 @@
 public class BeanImportTck {
 
     public static Test suite() {
-        try (BeanContext beanContext = BeanContext.run()) {
-            return Tck.testsFor(beanContext.getBean(Car.class), false, true);
-        }
+        BeanContext beanContext = BeanContext.run();
```

Comment:
I think we should close the `BeanContext`

https://github.com/micronaut-projects/micronaut-core/pull/7233